admin-dev/index.php

```diff
@@ -92,3 +92,5 @@
     // Prepare and trigger LEGACY admin dispatcher
     Dispatcher::getInstance()->dispatch();
 }
+
+// gdsgdsgs
```

Comment:
```suggestion
// gdsgdsgs
```

Could you please translate your comment to Russian, we only use this lang in order to comment

Reply:
Nice move @NeOMakinG